src/pages/content/Apology-to-australias-indigenous-peoples.md

```diff
@@ -0,0 +1,99 @@
+---
+path: "about-australia/our-country/our-people/apology-to-australias-indigenous-peoples"
```

Comment:
We should just keep the path paths as the content title for now so: `apology-to-australias-indigenous-peoples`